### DISCUSSION TITLE
Not able to pulish kata

### DISCUSSION THREAD

**Discussion by coderman1018** [codewars/codewars.com#3192] [General]
I have 5,020 honor points and I am on level 2 kyu on codewars, yet I am not able to create my own kata. Whenever I click on "New Kata" I am told "You do not have permissions to view this page". Why is this happening?

**Comment by hobovsky** [comment]
We would have to know your Codewars profile or user name to be able to check exactly. You can also jump into Codewars Discord and share a link to your Codewars profile in `#contact-mods`, and someone will check what happened.

**Reply by coderman1018** [reply]
My username is 'anushaj'.

**Comment by Kacarott** [comment]
Your account was restricted because you were cheating on many beta kata in order to downvote them. Not only is cheating itself against the rules, but being spiteful and trying to abuse the beta system in order to ruin the experience for others is extremely dishonorable and against the Code of Conduct.

**Reply by coderman1018** [reply]
Sorry, I wasn't cheating, I wanted to see the solution of the kata as I wasn't able to solve it and then test the solution out to understand how it worked. I downvoted because I found the description confusing and hard to understand. Please un-restrict my account as I will not do this again in the future.

**Reply by Kacarott** [reply]
You did this to 15 Kata in a row, and from the completion times you spent less than a minute on each one, so you didn't even try to understand the description, nor did you bother trying to understand the solutions as you claim. You also voted each kata as 8kyu, which shows you were clearly just trying to cause problems. Since you are lying here I have no reason to believe you will actually not do it again.

**Reply by coderman1018** [reply]
I was trying to find a solution I could understand. I am very sorry and if I do this again you can restrict me. Please un-restrict!
